### PR TITLE
feat: Added Figma plugin to export variables

### PR DESCRIPTION
**Pull request by tft-abel**
Wrote Figma plugin to export variables from Figma in the Design Token spec by W3C, a JSON format. A JSON file is created for each collection-theme permutation and grouped in a .zip file before it is downloaded.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,9 @@
-# figma-design-tokens
\ No newline at end of file
+# Export Design Tokens (W3C) from Figma
+
+This plugin exports Figma variables into separate JSON files per collection and modes grouped in a .zip file for easy download. 
+
+## Background
+This plugin has been created for managing the design system of the public mobility collaboration (Offtentlig Mobilitetssamarbeid [OMS]) in Norway. This project creates whitelabel software for the public transport sector, which can then be shared across the entire country. This means that multiple organisations are creating multiple themes using this design system. To manage the complexity, Figma variables are considered the source of truth for design decisions.
```
Comment (tfk-kaare) [inline README.md]: s/Offtentlig/Offentlig/

File: code.ts [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+  for (const collection of collections) {
+    object[idToKey[collection.id]] = await collectionAsJSON(idToKey, collection)
+  }
+
+  exportFiles(object)
+}
+
+/**
+ * Takes a tree of design tokens, split is up by collection and mode and generates a file structure that is send to a .zip file.
```
Comment (tfk-kaare) [inline code.ts]: 'split is up by …'? I guess it should be 'splits it up by …'

File: package.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "Design-Tokens-W3C-Export",
+  "version": "1.0.0",
```
Comment (tfk-kaare) [inline package.json]: Version 1.0.0? Already? No beta?

**Review by tfk-kaare — COMMENTED** [review]
- Coding style (eslint) should extend airbnb-base (package: eslint-config-airbnb-base). Unfortunately it doesn't cover comments/doc-blocks, but:
- Doc-blocks for functions should follow a more strict style. E.g. First line should stand alone and be no longer than 80 chars.  Following lines describe the function more in-depth and wraps at max 80 chars.
- eslint reports two errors (use of 'any' and 'collection' should be const)

I love the use of Typescript. It makes working with js a bit more sane :-)

File: package.json (hunk 3 above)
Comment (imable) [reply package.json]: Whoops, it was that by default, let's go for 0.1.0 instead, haha

**Comment by tft-abel** [comment]
Fixed @tft-kaare's comments by adding Airbnb base ESLint config and formalized the JSDoc comments

**Review by tfk-kaare — APPROVED** [review]
Let's do this!

**Pull request merged**